CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+Contributions to `ros2_rust` are very welcome!
+
+Work is coordinated in the [Matrix chat](https://matrix.to/#/+rosorg-rust:matrix.org). It's a good idea to check in with the other contributors there before starting development on a new feature.
+
+There are also occasional ROS 2 Rust WG meetings that are announced on the [ROS discourse forum](https://discourse.ros.org/).
+
+## Coding guidelines
+Aim for well documented and tested code.
+
+Use declarative sentences, not imperative sentences, for documenting functions. For instance, `Creates a by-value iterator.`, not `Create a by-value iterator`. 
+
+Use `cargo clippy` to help with writing idiomatic code.
+
+### Unsafe code
+Annotate every `unsafe` block with a `// SAFETY:` comment explaining why its safety preconditions are met.
```

Comment:
Should we also mention that `unsafe` blocks are to be kept as limited in size as possible? I don't know that `unsafe` programming is very common in Rust, so newer contributors might not know best practices for it.

We can also link to [The Rustonomicon](https://doc.rust-lang.org/nomicon/) for some guidance to new `unsafe` programmers, if that would be useful.

Reply:
Done. I didn't link to the Rustonomicon because then I'd feel like I need to link against every resource that's equally important as the Rustonomicon.